### ISSUE TITLE
subclass immediateSubclass contradiction

### ISSUE THREAD

**Issue by fcbr** [ontologyportal/sumo#9]
Looks like a pattern is emerging.  Similar situation to #7 and #8.

```
(instance subclass BinaryPredicate)
(subrelation immediateSubclass subclass)
```

but:

```
(subclass BinaryPredicate InheritableRelation)
(instance subclass PartialOrderingRelation)
(subclass PartialOrderingRelation TransitiveRelation)
```

this we conclude that:

```
(instance immediateSubclass TransitiveRelation)
```

however:

```
(instance immediateSubclass IntransitiveRelation)
```

and:

```
(disjoint TransitiveRelation IntransitiveRelation)
```

Contradiction.

**Comment by apease** [comment]
subrelation statement was previously removed, closing issue

**Comment by arademaker** [comment]
Solved in 89d8923.
